Mathlib/Order/InitialSeg.lean

```diff
@@ -44,8 +44,9 @@ any `b' < b` also belongs to the range). The type of these embeddings from `r` t
 `InitialSeg r s`, and denoted by `r ≼i s`.
 -/
 
-variable {α : Type*} {β : Type*} {γ : Type*} {r : α → α → Prop} {s : β → β → Prop}
-  {t : γ → γ → Prop}
+universe u
+
+variable {α γ : Type*} {β : Type u} {r : α → α → Prop} {s : β → β → Prop} {t : γ → γ → Prop}
```

Comment:
This looks like a regression. Can you please use explicit universes only where needed?